### DISCUSSION TITLE
Custom iOS app icons

### DISCUSSION THREAD

**Discussion by rodaimi** [community/community#3147] [Mobile] [answered]
I like the variety of built-in app icon designs available but I would like to be able to further customize the app icons.

It would be cool if I could add a custom background image overlayed by the GitHub logo with a custom color scheme.

**Comment by rnystrom** [comment, marked as answer]
Thanks for the feedback! Right now we're keeping the custom app icons pretty limited, but are exploring some new icons. We don't have any plans to add self-customizable icons however. If you have any ideas or suggestions for icons we should add, let me know!
